.github/pull_request_template.md

```diff
@@ -2,7 +2,7 @@ Please read [Github PR Conventions](https://github.com/haskell/cabal/blob/master
 
 ---
 
-**Template Α: This PR modifies `cabal` behaviour**
+**Template Α: This PR modifies `cabal` behaviour *or* `Cabal` API **
```

Comment:
What do we mean by `Cabal`? Is it just the `Cabal` package or any of the published packages?

Reply:
“`Cabal` the library and other libraries in `cabal` the repository”. I wanted to be less verbose as possible conveying this message, suggestions welcome!

Reply:
Does a markdown footnote work?

**Template Α: This PR modifies `cabal` behaviour *or* `Cabal`[^1] API **

[^1]: Cabal the library and other libraries in cabal the repository.

Reply:
In fact, changing the API of any of the 4 packages on Hackage requries a changelog entry and all 4 have an API (a library component). So maybe just "cabal behaviour or library API of any of the cabal packages". Cabal file format is covered in "cabal behaviour", I think.

Reply:
Current:

**Template Α: This PR modifies `cabal` behaviour *or* the API of any `cabal` package **

Would that do Phil?

Reply:
To keep it short, how about not describing what the API is here in the template and instead linking to another doc that describes it; in `CONTRIBUTING.md` or in the [docs](https://cabal.readthedocs.io/en/latest/)?

Reply:
I want to strike a balance between short and “immediately recognisable”. This is the latest iteration after some feedback on Matrix, your thoughts?

![image](https://github.com/haskell/cabal/assets/87714215/6b81a7d6-3a75-4aaf-a4cb-e9508a9eaf0f)

Reply:
I checked and there's already a (contributing changlog section](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#changelog).

Reply:
The template descriptions could be shorter with a link to CONTRIBUTING.md if that had definitions of behavioural changes we care about and what we consider to be the interface or API;

* Template A: This PR modifies \[behaviour or interface\](LINK-TO-DEFINITION)
* Template B: This PR doesn't modify \[behaviour or interface\](LINK-TO-DEFINITION)